package.json

```diff
@@ -29,15 +29,19 @@
     "@types/node-sass": "^4.11.1",
     "@types/react": "^17.0.0",
     "@types/react-dom": "^17.0.0",
-    "@typescript-eslint/eslint-plugin": "^4.8.2",
-    "@typescript-eslint/parser": "^4.8.2",
-    "eslint": "^7.14.0",
+    "@typescript-eslint/eslint-plugin": "^4.11.0",
+    "@typescript-eslint/parser": "^4.11.0",
+    "eslint": "^7.16.0",
+    "eslint-config-airbnb": "^18.2.1",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-jsx-a11y": "^6.4.1",
     "eslint-plugin-react": "^7.21.5",
+    "eslint-plugin-react-hooks": "^4.2.0",
     "husky": "^4.3.0"
   },
   "husky": {
     "hooks": {
       "pre-commit": "npm run lint-and-fix"
```

Comment:
In pre-commit, shouldn't the developer be shown what are the errors and warnings?
What would happen here is that ESLint would directly fix the formatting for the developer without the developer knowing there were any issues in the first place.

Reply:
- Dev doesn't need to be warned about extra spaces, missing semicolons, and errors which can be fixed automatically by `eslint`.
- If `eslint` makes the changes, files are changed and the developer needs to commit again.
- If there are any errors which `Eslint` cannot take care of, it'll reject the commit and show error messages.

Reply:
We once had a discussion around this exact thing on #dev-backend channel in RDS Discord server.
Link: https://discord.com/channels/673083527624916993/735598747949465640/765564963766140928

Reply:
@swarajpure Changes are done, please review.